lib/testpool/test-pool.js

```diff
@@ -0,0 +1,88 @@
+const path = require('path');
+const debug = require('debug')('ava:pool');
+const serializeError = require('../serialize-error');
+
+if (!process.env.NODE_ENV) {
+	process.env.NODE_ENV = 'test';
```

Comment:
This changes the current behavior. If `process.env.NODE_ENV === ''`, that should be preserved.